manifests/nexus.pp

```diff
@@ -0,0 +1,69 @@
+#
+# Download from Nexus using REST API
+# More info here: https://repository.sonatype.org/nexus-restlet1x-plugin/default/docs/path__artifact_maven_content.html
+#
+define archive::nexus (
+  $ensure       = present,
+  $checksum_type = 'md5',
+  $nexus_url    = undef,
+  $gav          = undef,
```

Comment:
Oh, that's fine if it's nexus convention. You don't have to repeat nexus in the parameter, so it might be better just to leave it as archive::nexus::url instead of archive::nexus::nexus_url.